DZSwiftUI/DZSwiftUI/DescriptionView.swift

```diff
@@ -102,6 +105,29 @@ struct DescriptionView: View {
             )
             .foregroundColor(.yellow)
             .padding(.bottom, Constants.furnitureImagePaddingBottom)
+            .aspectRatio(contentMode: .fit)
+            .animation(.easeInOut)
+            .offset(
+                x: descriptionViewModel.positionDescriptionInfo.width + startDragOffsetValue.width,
+                y: descriptionViewModel.positionDescriptionInfo.height + startDragOffsetValue.height
+            )
+            .scaleEffect(descriptionViewModel.scaleDescriptionInfo)
+            .gesture(
+                MagnificationGesture()
+                    .onChanged { value in
+                        descriptionViewModel.scaleDescriptionInfo = value
+                    }
+            )
+            .gesture(
+                DragGesture()
+                    .updating($startDragOffsetValue, body: { value, state, _ in
+                        state = value.translation
+                    })
+                    .onEnded { value in
+                        self.descriptionViewModel.positionDescriptionInfo.height += value.translation.height
+                        self.descriptionViewModel.positionDescriptionInfo.width += value.translation.width
+                    }
```

Comment:
```suggestion
                DragGesture()
                    .updating($startDragOffsetValue, body: { value, state, _ in
                        state = value.translation
                    })
                    .onEnded { value in
                        self.descriptionViewModel.positionDescriptionInfo.height += value.translation.height
                        self.descriptionViewModel.positionDescriptionInfo.width += value.translation.width
                    }
```
думаю это можно завернуть в свойство и вынести
private var dragGesture: some Gesture {
код сюда
}